src/golive/cdn.md

```diff
@@ -52,3 +52,61 @@ When using a CDN the Platform.sh router's HTTP cache becomes redundant.  In most
         # Disable the HTTP cache on this route. It will be handled by the CDN instead.
         enabled: false
 ```
+
+## Secure/hide your project from being accessed directly
+
+When using a CDN, you might not want users to access your platform.sh origin directly. There are 2 ways to secure your origin.
+
+### 1 Basic HTTP Authentication
+
+#### Password protected
+You can password protect your project using [HTTP access control](https://docs.platform.sh/administration/web/configure-environment.html#http-access-control). 
+
+Make sure that you generate a password of sufficient strength. You can then use share the password with your CDN provider. Make sure the CDN adds a header to authenticate correctly to your origin.
+
+
+Simply add a custom header to the origin request with the base64 encoded username:password.
+
+`Authorization: Basic QWxhZGRpbjpPcGVuU2VzYW1l` 
+
+
+
+#### IP whitelisting
+
+If your CDN does not support adding headers to the request to origin, you can allow the IP addresses of your CDN.
+
+*Note: you WILL have to update your configuration when your cdn updates their ip addresses*
+
+List of ip ranges for:
+
+- [CloudFlare](https://www.cloudflare.com/ips/)
+- [Fastly](https://docs.fastly.com/en/guides/accessing-fastlys-ip-ranges)
+
+*For CloudFlare, using the HTTP access control described above is the recommended way of securing your origin.*
+
+### 2 Client authenticated TLS
+If your CDN offers this option, an alternative way of securing the connection is [client authenticated TLS](/configuration/routes/https.html#client-authenticated-tls).
+
+*Note: Please remember to permit your developers to access the origin by creating your own certificate else they won't be able to access the project url directly. (see below)*
```

Comment:
```suggestion
> **note**
> Please remember to permit your developers to access the origin by creating your own certificate else they won't be able to access the project url directly. (see below)
```

Reply:
This is the note format we use.  Please switch the other notes in this PR to this format.